lib/thrift/generator/server/binary_framed.ex

```diff
@@ -94,6 +95,7 @@ defmodule Thrift.Generator.Server.BinaryFramed do
       Thrift.TApplicationException ->
         quote do
           unhandled ->
+            Logger.error("Unhandled exception: #{Exception.message(unhandled)}")
```

Comment:
Should there be some context hint in this?  `"Unhandled TApplicationException"` or something?